src/template/commands/scaffold.rs

```diff
@@ -12,7 +14,7 @@ pub fn part_two(input: &str) -> Option<u32> {
     None
 }
 
-advent_of_code::main!(DAY);
+advent_of_code::main!(DAY_NUMBER);
 
 #[cfg(test)]
 mod tests {
```

Comment:
The unit tests still have references to `DAY` which should be `DAY_NUMBER` now.

Reply:
`advent_of_code::main!` creates a const value `DAY` of type `Day`.
The `DAY` in unit tests refer to this const.

Reply:
Gotcha, missed that.